core/chains/evm/txmgr/orm.go

```diff
@@ -232,6 +233,20 @@ func (o *orm) FindEthTxWithAttempts(etxID int64) (etx EthTx, err error) {
 	return etx, errors.Wrap(err, "FindEthTxWithAttempts failed")
 }
 
+func (o *orm) FindEthTxAttemptConfirmedByEthTxIDs(ids []int64) (etx []EthTxAttempt, err error) {
```

Comment:
nit/ consider omitting the return names here since they aren't used and don't add any info.
```suggestion
func (o *orm) FindEthTxAttemptConfirmedByEthTxIDs(ids []int64) ([]EthTxAttempt, error) {
```